fix(guide): clarify sharing docker compose (#1789)
TRaSH-Guides/Guides: docs/Misc/how-to-provide-a-docker-compose.md

!!! warning
    Please don't use screenshots to provide the info as they aren't always revealing all the needed info

> Use one of the following from terminal (i.e. puTTy) to generate the docker compose with or without the use of `sudo`. Note you can pass in multiple container names.

Reviewer: the first part of this sentence makes no sense to me

Reviewer: ```suggestion
Use one of the following commands in the terminal of your choice (e.g. puTTY) to generate the docker compose, with or without the use of `sudo`. Note that you can pass in multiple container names.
```
